docs/data/joy/customization/using-css-variables/using-css-variables.md

```diff
@@ -1,81 +1,97 @@
 # Using CSS variables
 
-<p class="description">Learn how to use CSS variables to customize Joy UI's components.</p>
+<p class="description">Learn how to use CSS variables to customize Joy UI components.</p>
 
-## Theme object
+## Introduction
 
-The `CssVarsProvider` reads the theme input (or the default theme) and creates the CSS variables according to the theme structure.
-It also creates an object that refers to the generated CSS variables under `theme.vars` so that you can use them from the JavaScript theme object.
+CSS variables are created by wrapping your app with the CssVarsProvider and using the `theme.vars` notation, allowing for an efficient customization experience.
 
-The `theme.vars` is available in all styling APIs that Joy UI offers:
+```jsx
+import { CssVarsProvider } from '@mui/joy/styles';
+
+function App() {
+  return <CssVarsProvider>...</CssVarsProvider>;
+}
+```
 
-1. The `styled` function
+## Usage
+
+### Styling APIs
+
+Use the `theme.vars` notation with any styling APIs supported by Joy UI:
+
+#### styled function
 
 ```js
 const Div = styled('div')(({ theme }) => ({
-  // The result is 'var(--joy-palette-primary-500)'
+  // Outputs 'var(--joy-palette-primary-500)'
   color: theme.vars.palette.primary[500],
 }));
 ```
 
-2. The `sx` prop
+#### sx prop
 
 ```jsx
-// The result is 'var(--joy-shadow-sm)'
+// Outputs 'var(--joy-shadow-sm)'
 <Chip sx={(theme) => ({ boxShadow: theme.vars.shadow.sm })} />
 ```
 
-3. Style overrides (themed components)
+You can also use a short-hand syntax to resolve the values from the `theme.vars.*` the same way the example above does.
+
+```js
+<Chip
+  sx={{
+    border: '1px solid',
+
+    // For color related properties, lookup from `theme.vars.palette`
+    color: 'neutral.800', // 'var(--joy-palette-neutral-800)'
+    borderColor: 'neutral.400', // 'var(--joy-palette-neutral-400)'
+
+    // lookup from `theme.vars.shadow`
+    shadow: 'sm', // 'var(--joy-shadow-sm)'
+
+    // lookup from `theme.vars.fontSize`
+    fontSize: 'sm', // 'var(--joy-fontSize-sm)'
+  }}
+/>
+```
+
+#### Themed components
 
 ```jsx
 extendTheme({
   components: {
     JoyButton: {
       root: ({ theme }) => ({
-        // The result is 'var(--joy-fontFamily-display)'
+        // Outputs 'var(--joy-fontFamily-display)'
         fontFamily: theme.vars.fontFamily.display,
       }),
     },
   },
 });
 ```
 
-4. The `useTheme` hook.
+#### useTheme hook
 
-### Alpha channel colors
+Note that the `useTheme` hook should be used in components wrapped by the `CssVarsProvider` component.
```

Comment:
Isn't this always the case? My understanding is that this note applies equally to everything on the page.